src/Database/Reader.php

```diff
@@ -110,6 +110,26 @@ public function anonymousIp($ipAddress)
     }
 
     /**
+     * This method returns a GeoIP2 ASN model.
+     *
+     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     *
+     * @return \GeoIp2\Model\Asn
+     *
+     * @throws \GeoIp2\Exception\AddressNotFoundException if the address is
```

Comment:
@wesrice I'm following prior art with this pull request.  I shall defer to @oschwald on the exception testing. :)